Sources/TuistCoreTesting/MetadataProviders/MockOtoolController.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+import RxSwift
+import TSCBasic
+@testable import TuistCore
+
+final class MockOtoolController: OtoolControlling {
```

Comment:
I recommend generating the mocks with [this utility](https://github.com/seanhenry/SwiftMockGeneratorForXcode) that we have started to use to generate mocks in the project.